tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/PetStoreOperations.java

```diff
@@ -54,6 +59,27 @@
     return client.getPets(ownerName, config);
   }
 
+  @MediaType(TEXT_PLAIN)
+  public InputStream getStreamedSignature(String signature) {
+    return new ByteArrayInputStream(signature.getBytes());
+  }
+
+  public List<String> getPetsWithIntermitentConnectionProblemAndClosingStream(@Connection PetStoreClient client,
```

Comment:
Just to be sure that the reconnection is done, could you return some value, like a flag telling that the reconnection has been executed, just to be sure that if in some moment the test is changed and no reconnection is done the test will be still 🍏

Reply:
adding